chemicalx/models/epgcnds.py

```diff
@@ -21,18 +21,21 @@ class EPGCNDS(Model):
        <https://ojs.aaai.org/index.php/AAAI/article/view/7236>`_
     """
 
-    def __init__(self, *, in_channels: int, hidden_channels: int = 32, out_channels: int = 16):
+    def __init__(
+        self, *, in_channels: int, hidden_channels: int = 32, middle_channels: int = 16, out_channels: int = 1
```

Comment:
Consider adding a default value for in_channels?

Reply:
Yes, why not!